### DISCUSSION TITLE
Does 3.2.4 apply to content order/layout?

### DISCUSSION THREAD

**Discussion by Swinejamin** [w3c/wcag#3397] [Q&A] [answered]
I have two elements that are reused across multiple pages of a site. One is a user info card with profile image & links. The other is a simple contact form. Each element in isolation meets all other accessibility requirements on its own, and tabbing order is not changed from default. The only difference from page to page is that the two elements sometimes appear in a different order in the document outline, e.g. user card first on the homepage, but at the end of main content on other pages. 

As I read 3.2.4, this situation (content elements appearing in a different order across pages) does not fail the SC, because everything within them is clearly identified. Does 3.2.4 consider ordering/layout to be a factor in consistent identification?

**Comment by patrickhlauke** [comment, marked as answer]
> Does 3.2.4 consider ordering/layout to be a factor in consistent identification?

No, as you can see by the fact that this isn't mentioned in https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html
